### PR TITLE
feat(receiver-mock): add dump endpoint

### PR DESCRIPTION
**Pull request by sumo-drosiek**
add endpoint which just dumps requests (to test e.g. collectd plugin)

### CODE REVIEW

File: src/rust/receiver-mock/src/router/mod.rs [hunk 1]
```diff
@@ -577,6 +577,12 @@ pub async fn handler_logs_count(
     HttpResponse::Ok().json(LogsCountResponse { count })
 }
 
+pub async fn handler_dump(body: web::Bytes) -> impl Responder {
```
Comment (pmalek-sumo) [inline src/rust/receiver-mock/src/router/mod.rs:580]: I wonder if there is no other type that can be used as an argument here to get the stringified body already at the argument level.

**Review by pmalek-sumo — APPROVED** [review]

File: src/rust/receiver-mock/src/router/mod.rs (hunk 1 above)
Comment (swiatekm-sumo) [reply src/rust/receiver-mock/src/router/mod.rs:580]: I couldn't see any obvious way of doing this in the actix docs.

**Review by swiatekm-sumo — COMMENTED** [review]
Can you document this in the README?

File: src/rust/receiver-mock/src/router/mod.rs (hunk 1 above)
Comment (sumo-drosiek) [reply src/rust/receiver-mock/src/router/mod.rs:580]: Me neither: https://docs.rs/actix-web/latest/actix_web/web/index.html

**Review by pmalek-sumo — APPROVED** [review]

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**